changelog.d/735.misc

```diff
@@ -0,0 +1 @@
+Fix lint errors and run CI on all PRs.
```

Comment:
N.B. if this PR is fixing an issue with another PR, it probably doesn't need to appear in the changelog. What you can do is paste the same changelog you gave to the previous PR into this file, and it will automerge at release time.